libnopegl/src/api.c

```diff
@@ -424,6 +424,7 @@ static const char *get_cap_string_id(unsigned cap_id)
     case NGL_CAP_MAX_TEXTURE_DIMENSION_2D:      return "max_texture_dimension_2d";
     case NGL_CAP_MAX_TEXTURE_DIMENSION_3D:      return "max_texture_dimension_3d";
     case NGL_CAP_MAX_TEXTURE_DIMENSION_CUBE:    return "max_texture_dimension_cube";
+    case NGL_CAP_TEXTLIBS:                      return "textlibs";
```

Comment:
`NGL_CAP_TEXTLIBS` / "textlibs" is, IHMO, not really explicit about what it enables. Would something like `NGL_CAP_TEXT_RENDERING` work ?

Reply:
We do actually have text rendering without external text libraries, so `NGL_CAP_TEXT_RENDERING` wouldn't work.

In general "rendering" doesn't work here, the keyword is libraries, as in "external libraries". `NGL_CAP_TEXT_EXTERNAL_LIBRARIES`?

Reply:
The idea behind my remark  is that I don't think the end user cares about whether we rely on external libraries or not. He just want to know if the feature that will let him use his fonts is enabled or not. While we indeed support rendering of the builtin font, I don't think it is relevant to do some motions/effects with it.
That being said, `NGL_CAP_TEXT_EXTERNAL_LIBRARIES` is better than `NGL_CAP_TEXTLIBS`.

Reply:
`NGL_CAP_TEXT_FONTS`?

Reply:
After an offline discussion, we decide to go with `NGL_CAP_TEXT_LIBRARIES`.